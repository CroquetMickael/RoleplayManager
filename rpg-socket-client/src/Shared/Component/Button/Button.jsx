import React from "react";

const Button = (props) => (
  <button
    className="w-full p-1 my-1 text-white bg-blue-400 rounded-sm hover:bg-blue-600"
    onClick={props.onClick}
  >
    {props.children}
  </button>
);

export { Button };
