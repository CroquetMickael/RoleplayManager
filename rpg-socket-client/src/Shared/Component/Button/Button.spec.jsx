import React from "react";
import { render } from "@testing-library/react";
import { Button } from "./Button";

it("renders correctly", () => {
  const tree = render(<Button>button children :)</Button>).container;
  expect(tree).toMatchSnapshot();
});
