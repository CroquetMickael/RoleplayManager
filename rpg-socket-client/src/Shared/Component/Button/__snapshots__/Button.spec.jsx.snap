// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <button
    class="w-full p-1 my-1 text-white bg-blue-400 rounded-sm hover:bg-blue-600"
  >
    button children :)
  </button>
</div>
`;
